rules/web/webserver_generic/web_xss_in_access_logs.yml

```diff
@@ -33,6 +34,9 @@ detection:
         - '${alert'
         - 'javascript:alert'
         - 'javascript%3Aalert'
+        - 'prompt'
+        - 'confirm'
+        - '\u'
```

Comment:
The prompt and confirm can be very common in URLs so they can' t be used on their own.

As for the `\u` its not indicative of XSS on its own.

Reply:
@ThureinOo you need to provide better strings as all the strings in this PR are not accepted for this PR to be merged.